### PR TITLE
Dom controller interface

### PR DESCRIPTION
**Pull request by mastermind-mayhem**
Just looking for review, 
PLEASE DON'T MERGE WITHOUT APPROVAL

Point of this PR is to make it able for the drivers to switch their controller layouts through a match on the dashboard
the code should create a drop down menu on the dashboard and then reassign values to constants causing different behaviors on the controllers

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -4,13 +4,22 @@
...
-import frc.robot.commands.teleopCmd;
+import frc.robot.commands.basic.autoCmd;
+import frc.robot.commands.basic.teleopCmd;
 import frc.robot.subsystems.drivetrainSubsystem;
 
 public class RobotContainer {
+  // Defining Controllers
+  private final Joystick driver1 = new Joystick(Constants.DRIVER_ONE_PORT);
+  private final Joystick driver2 = new Joystick(Constants.DRIVER_SEC_PORT);
+  private final Joystick operator = new Joystick(Constants.OPER_PORT);
```
Comment (mathyourlife) [inline src/main/java/frc/robot/RobotContainer.java:17]: These don't look like they're being used. (compared to the ones in teleopCmd.java)

File: src/main/java/frc/robot/commands/basic/teleopCmd.java [hunk 2]
```diff
@@ -11,33 +11,40 @@ public class teleopCmd extends CommandBase {
...
-      new SlewRateLimiter(Constants.SLEWTURN); // from 2 to 2.5 to 3.5 to 4.0 to 4.5
+  private double turns;
+  private double speeds;
+  private final Joystick driver1 = new Joystick(Constants.DRIVER_ONE_PORT);
+  private final Joystick driver2 = new Joystick(Constants.DRIVER_SEC_PORT);
+  private final SlewRateLimiter slrForTurn = new SlewRateLimiter(Constants.SLEWTURN);
   private final SlewRateLimiter slrForDrive = new SlewRateLimiter(Constants.SLEWSPEED);
 
-  private final Joystick stick = new Joystick(Constants.DRIVER_PORT);
   private boolean isTeleOp = false;
```
Comment (mathyourlife) [inline src/main/java/frc/robot/commands/basic/teleopCmd.java:21]: `isTeleOp` variable is unused.

File: src/main/java/frc/robot/commands/basic/autoCmd.java [hunk 3]
```diff
@@ -2,7 +2,7 @@
 // Open Source Software; you can modify and/or share it under the terms of
 // the WPILib BSD license file in the root directory of this project.
 
-package frc.robot.commands;
+package frc.robot.commands.basic;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
```
Comment (mathyourlife) [inline src/main/java/frc/robot/commands/basic/autoCmd.java:8]: Can you change the name of this file to `AutoCmd.java`  (and update the class to match)

Same with `teleopCmd.java`, `controllerSubsystem.java`, and `drivetrainSubsystem.java`

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (mastermind-mayhem) [reply src/main/java/frc/robot/RobotContainer.java]: They are set aside for when the actual code is constructed, just a placeholder

File: src/main/java/frc/robot/commands/basic/teleopCmd.java (hunk 2 above)
Comment (mastermind-mayhem) [reply src/main/java/frc/robot/commands/basic/teleopCmd.java:22]: it isn't, down at the bottom of the file it uses it to determine when to stop the program

**Pull request closed without merging**